.github/workflows/nomad.yaml

```diff
@@ -283,15 +283,19 @@ jobs:
           RUST_LOG: debug
         run: nix develop --command cargo run --release --bin holochain-summariser
 
+      - name: Generate summary id for artifacts
```

Comment:
Ensure consistent naming between summary visualizer html and summary json artifacts